backend/licenses-core/src/main/java/org/eclipse/sw360/licenses/db/LicenseTypeRepository.java

```diff
@@ -37,7 +38,10 @@ public LicenseTypeRepository(DatabaseConnectorCloudant db) {
     }
 
     public List<LicenseType> searchByLicenseType(String name) {
-        return queryByPrefix("bylicensetype", name);
+        List<LicenseType> all = queryByPrefix("bylicensetype", ""); // get all entries (or a broad range)
```

Comment:
Please use appropriate function:
```suggestion
        List<LicenseType> all = queryView("bylicensetype"); // get all entries (or a broad range)
```

Reply:
Comment addressed.

Reply:
Comment addressed.